test/select/bootstrap-select.spec.js

```diff
@@ -0,0 +1,118 @@
+describe('pf-bootstrap-select', function () {
+
+  var $scope, $compile;
+
+  beforeEach(module('patternfly.select'));
+
+  beforeEach(inject(function (_$rootScope_, _$compile_, _$timeout_) {
+    $scope = _$rootScope_;
+    $compile = _$compile_;
+    $timeout = _$timeout_;
+  }));
+
+  describe('Page with pf-select directive', function () {
+
+    var compileSelect = function (markup, scope) {
+      var el = $compile(markup)(scope);
+      scope.$digest();
+      return el;
+    };
+
+    it('should generate correct options from ng-options', function () {
+
+      $scope.options = ['a','b','c'];
+      $scope.modelValue = $scope.options[1];
+
+      var select = compileSelect('<select pf-bootstrap-select ng-model="modelValue" ng-options="o as o for o in options"></select>', $scope);
+
+      $timeout.flush();
```

Comment:
Is this necessary? I don't see a timeout being used.

Reply:
Hmm...I removed the timeouts but the unit tests failed.  Could it be due to the operations being run in post-link?

Reply:
Maybe the bootstrap code is using a timeout.

Reply:
or it could be the $applyAsync